src/ssh_session.ts

```diff
@@ -56,8 +56,8 @@ export class SSHSession extends BaseSession {
         }).then(args => {
                 this.onSSHSession(args.session)
         }).catch(e => {
-                console.log("SSH key startSession failed", e.toString())
                 const msg = e.toString()
+                console.log("SSH key startSession failed", msg)
```

Comment:
Please, either use terminal7.log or remove

Reply:
Wanted it to be consistent with passConnect

Reply:
I guess passConnect is wrong too... terminal7.log let's the user copy to log to the clipboard.

Reply:
Alright